nixos/modules/services/x11/redshift.nix

```diff
@@ -56,23 +56,50 @@ in {
 
     brightness = {
       day = mkOption {
-        type = types.str;
-        default = "1";
+        type = types.float;
+        default = 1.0;
```

Comment:
This is also technically a breaking change. I can document it in release notes if necessary.